src/styles/main.scss

```diff
@@ -1,3 +1,41 @@
+@import "./header.scss";
+@import "./card.scss";
+@import "./stars.scss";
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4,200px);
```

Comment:
Would be better to use variable for your card width instead of 200px because it will be easier to maintain the code